modules/core/src/main/java/org/apache/ignite/internal/binary/BinaryObjectExImpl.java

```diff
@@ -240,6 +242,23 @@ private String toString(BinaryReaderHandles ctx, IdentityHashMap<BinaryObject, I
         return buf.toString();
     }
 
+    /** */
+    private Object fieldForToString(BinaryReaderHandles ctx, String name) {
+        try {
+            return field(ctx, name);
+        }
+        catch (Exception e) {
+            OptimizedMarshallerInaccessibleClassException e1 =
+                X.cause(e, OptimizedMarshallerInaccessibleClassException.class);
+
+            String msg = "Failed to create a string representation";
+
+            return e1 != null
+                ? "(" + msg + ": class not found " + e1.inaccessibleClass() + ")"
```

Comment:
No `;` for this case?

Reply:
```java
? "(" + msg + ": class " + e1.inaccessibleClass() + " not found);"
```

Reply:
I suppose there is no `;` in both cases. There are 2 options for a message here:
* `(Failed to create a string representation: class not found ExternalTestClass1)`
* `(Failed to create a string representation)`